README.md

```diff
@@ -114,25 +114,25 @@ A browser implementation of this functionality would differ in two ways:
 
 ## The SDK and the API
 
-The Adobe Experience Platform Launch API is a RESTful
+The Adobe Experience Platform Tags API (Reactor API) is a RESTful
```

Comment:
This should change to `The Adobe Experience Platform Reactor API`. The SDK supports Launch Server Side, and we'll likely have some talks about it supporting new APIs as well.